phoenix-core/src/main/java/org/apache/phoenix/coprocessor/UngroupedAggregateRegionScanner.java

```diff
@@ -479,6 +486,8 @@ void deleteCForQ(Tuple result, List<Cell> results, UngroupedAggregateRegionObser
             delete.addColumns(deleteCF,  deleteCQ, ts);
             // force tephra to ignore this deletes
             delete.setAttribute(PhoenixTransactionContext.TX_ROLLBACK_ATTRIBUTE_KEY, new byte[0]);
+            // TODO: should we set the source of operation attribute here.
+            // TODO: This will be true if we drop column or drop table
```

Comment:
@gjacoby126  Could you please advise whether should we add the attribute here ? This codepath is used when we drop the column which will in turn generate Delete Marker.